<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HempicGUI</class>
 <widget class="QMainWindow" name="HempicGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HempicGUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/iconos/resources/iconos/Hempic.ico</normaloff>:/iconos/resources/iconos/Hempic.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>184</width>
      <height>376</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QComboBox" name="periodo7Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="periodo1Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="periodo6Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QComboBox" name="periodo10Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="comentarioP1">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="periodo5Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QComboBox" name="periodo9Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="periodo3Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="periodo4Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="periodo2Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QComboBox" name="periodo8Combo">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="comentarioP2">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="comentarioP3">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="comentarioP4">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QToolButton" name="toolButton_5">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QToolButton" name="comentarioP6">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QToolButton" name="comentarioP7">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QToolButton" name="comentarioP8">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QToolButton" name="comentarioP10">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QToolButton" name="comentarioP9">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/comments.png</normaloff>:/iconos/resources/iconos/comments.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="configuracionBoton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>27</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Configuración</string>
    </property>
    <property name="statusTip">
     <string>Configuración</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/iconos/resources/iconos/cog.png</normaloff>:/iconos/resources/iconos/cog.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>40</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>9:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>11:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>12:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>13:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>14:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>15:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>16:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>17:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>18:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>144</width>
      <height>97</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="diaIntLabel">
         <property name="font">
          <font>
           <family>Ubuntu Condensed</family>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="statusTip">
          <string>Día del mes</string>
         </property>
         <property name="text">
          <string>10</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="diaStringLabel">
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="statusTip">
            <string>Día de la semana</string>
           </property>
           <property name="text">
            <string>Miercoles</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="mesLabel">
             <property name="font">
              <font>
               <family>Ubuntu Condensed</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="statusTip">
              <string>Mes</string>
             </property>
             <property name="text">
              <string>Dic</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="anioLabel">
             <property name="font">
              <font>
               <family>Ubuntu Condensed</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="statusTip">
              <string>Año</string>
             </property>
             <property name="text">
              <string>2005</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="cambiarFechaBoton">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="statusTip">
        <string>Cambiar fecha</string>
       </property>
       <property name="text">
        <string>Cambiar Fecha</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/iconos/resources/iconos/calendar.png</normaloff>:/iconos/resources/iconos/calendar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>253</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
